hw/top_earlgrey/data/ip/chip_otp_ctrl_testplan.hjson

```diff
@@ -0,0 +1,208 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_otp_ctrl
+  testpoints: [
+    // OTP (pre-verified IP) integration tests:
+    {
+      name: chip_otp_ctrl_init
+      desc: '''Verify the OTP ctrl initialization on chip power up.
+
+            Verify that the chip powers up correctly on POR.
+            - The pwrmgr initiates a handshake with OTP ctrl and later, with LC ctrl in subsequent
+              FSM states. Ensure that the whole power up sequence does not hang.
+            - Verify with connectivity assertion checks, the handshake signals are connected.
+            - Ensure that no interrupts or alerts are triggered.
+            '''
+      features: ["OTP_CTRL.INIT"]
+      stage: V2
+      si_stage: SV1
+      tests: ["chip_sw_lc_ctrl_transition"]
+    }
+    {
+      name: chip_sw_otp_ctrl_keys
+      desc: '''Verify the proliferation of keys to security peripherals.
+
+            - Verify the correctness of keys provided to SRAM ctrl (main & ret), flash ctrl, keymgr,
```

Comment:
This may not be testable in silicon/fpga, so add a `SiVal` comment at the bottom of the description.

Reply:
Yes all these (subsequent few comments) can be properly address once we have manufacturing test plan is available.